src/base/rss/rssdownloadrule.h

```diff
@@ -88,6 +88,9 @@ namespace Rss
         bool operator==(const DownloadRule &other) const;
 
     private:
+        bool matches(const QString &articleTitle, QString expression) const;
```

Comment:
I'm not going to argue more on this (since it's trivial & pointless), but my compiler (gcc 6.3.1) can take my suggested code.